Tests/FluentAssertions.Specs/Primitives/StringAssertionSpecs.Be.cs

```diff
@@ -346,8 +346,8 @@ public void Mismatches_in_multiline_text_includes_the_line_number()
             act.Should().Throw<XunitException>().WithMessage($"""
                 Expected subject to be the same string, but they differ on line 5 and column 16 (index {expectedIndex}):
                              ↓ (actual)
-                  "…-> Bob : Another…"
-                  "…-> Bob : Invalid…"
+                  "…-> Bob : Another authentication Request*\nAlice <-- Bob :…"
+                  "…-> Bob : Invalid authentication Request\r\nAlice <-- Bob :…"
```

Comment:
```suggestion
                  "…-> Bob : Invalid authentication Request*\nAlice <-- Bob :…"
```
Uhh... missed that one 🤦‍♂️